app/scripts/src/content/content-controller.js

```diff
@@ -31,8 +31,8 @@ ContentController.prototype = {
   },
 
   onSearchError: function(status, response) {
-    this.sendAnalyticsEvent({ name: 'onSearchError', value: status });
-    console.error('traktflix: Search error', status, response);
+    this.sendAnalyticsEvent({ name: 'onSearchError', value: status + (this.item && this.item.title) ? this.item.title : "Invalid item title" });
```

Comment:
I think you can call `this.item.title` directly here. If the item is `undefined` it will probably break before gets here.